data/creaturescripts/scripts/regenerate_stamina.lua

```diff
@@ -14,9 +14,9 @@ function onLogin(player)
 	local staminaMinutes = player:getStamina()
 	local maxNormalStaminaRegen = 2340 - math.min(2340, staminaMinutes)
 
-	local regainStaminaMinutes = offlineTime / 180
+	local regainStaminaMinutes = offlineTime / configManager.getNumber(configKeys.STAMINA_REGEN_MINUTE)
 	if regainStaminaMinutes > maxNormalStaminaRegen then
-		local happyHourStaminaRegen = (offlineTime - (maxNormalStaminaRegen * 180)) / 600
+		local happyHourStaminaRegen = (offlineTime - (maxNormalStaminaRegen * 180)) / configManager.getNumber(configKeys.STAMINA_REGEN_PREMIUM)
```

Comment:
shouldn't this 180 and the values at line 8 and 10 be changed too?

Reply:
its required offline time to start regenerate stamina, i can do this configurable too